texts/epn_paper.md

```diff
@@ -0,0 +1,74 @@
+# Let's talk online
+
+<!-- #### COVID lockdown disrupted everyone's work, travel, and likely mean that you attended an online talk. -->
+
+Your routine has changed drastically over the course of the last couple of months due to the lockdowns following the novel coronavirus outbreak.
+You now mostly stay at home, hopefully safe and healthy.
+Your classes are now taught online, and you probably had to quickly figure out how to run them at all.
+If you do experiments, your lab is either closed or operates at a much lower capacity.
+Your travel plans are cancelled. Still, despite all this, you have likely attended a scientific presentation or even gave one yourself.
+The only difference is that it happened online, perhaps as part of a virtual conference.
+
+<!-- #### Online talks were gaining momentum already before, and for a good reason. -->
+
+Virtual conferences were gaining momentum even before the coronavirus crisis due to multiple advantages over traditional in-person gatherings.
+With the increasing concerns about greenhouse emissions, many prefer to limit their long-distance travel.
+For some community members with limited mobility, either for family, health, or budget reasons, traditional conferences are off limits altogether.
```

Comment:
```suggestion
For some community members with limited mobility, either for family, health, or budget reasons, traditional conferences are off limits altogether, and the online events are the only available option.
```

Reply:
```suggestion
For some community members with limited mobility, either for family, health, or budget reasons, traditional conferences are off limits altogether and online events are the only available option.
```

Reply:
This is a composite sentence. IIRC those always need a comma.

Reply:
Oh sorry, I meant to only remove the 'the'. Thanks for spotting that